pkg/crc/machine/machine.go

```diff
@@ -91,6 +92,13 @@ func Start(startConfig StartConfig) (StartResult, error) {
 
 	// Pre-VM start
 	driverInfo, _ := getDriverInfo(startConfig.VMDriver)
+	if driverInfo.DriverPath != "" {
+		/* We need to add the directory containing crc-driver-xxx to the PATH
+		* so that libmachine can find it
+		*/
+		os.Setenv("PATH", fmt.Sprintf("%s:%s", driverInfo.DriverPath, os.Getenv("PATH")))
+		logging.InfoF("machine.Start(): driverInfo.DriverPath=%s PATH=%s", driverInfo.DriverPath, os.Getenv("PATH"))
```

Comment:
So here we are explicit setting the driver path to env. I though we make changes in libmachine side to take this driver path as argument and run it.

Reply:
Have a look at: https://github.com/code-ready/machine/commit/59879005e2d35019745338a9d8e364ad6dd86bba

Reply:
changing this on libmachine side is quite invasive as for now its API is based on machine names only. So I went for this approach for now as it's probably simpler. I'll look at changing libmachine API instead.

Reply:
This is now done, see https://github.com/code-ready/machine/pull/13 
And the end result is actually simpler than what I was doing here with PATH changes.